apps/prairielearn/elements/pl-file-preview/pl-file-preview.mustache

```diff
@@ -26,17 +26,20 @@
         </div>
         <div class="alert alert-danger mt-2 d-none error" role="alert"></div>
         <div class="file-preview collapse" id="file-preview-contents-{{uuid}}-{{index}}">
-          <img class="mw-100 mt-2 d-none">
-          <pre class="bg-dark text-white rounded p-3 mt-2 mb-0 d-none"><code></code></pre>
+          <img class="d-none mw-100 mt-2">
+          <div class="file-preview-container mt-2">
```

Comment:
This was changed in the interest of separating the expansion behavior from whether we're displaying a plaintext or rendered notebook preview.